docs/guides/migrating-from-camunda-platform-7/adjusting-dmn-models.md

```diff
@@ -0,0 +1,20 @@
+---
+id: adjusting-dmn-models
+title: Adjusting DMN models
+description: "Learn how to adjust your DMN models when migrating from Camunda Platform 7 to Camunda Platform 8."
+---
+
+For Camunda Platform 8, [a former community extension](https://github.com/camunda-community-hub/dmn-scala), built by core Camunda developers is productized. This engine has a higher coverage of DMN elements. This engine can execute DMN models designed for Camunda Platform 7. However, there are some small differences as outlined in this document.
+
+You can use the tooling mentioned above to convert your DMN models from Camunda Platform 7 to Camunda Platform 8.
```

Comment:
What tooling specifically are we referring to?

Reply:
@ingorichtsmeier, you can use the modeler converter for DMN models too, right? The README is a bit unclear here - https://github.com/camunda-community-hub/camunda-7-to-8-migration/tree/main/modeler-plugin-7-to-8-converter

Reply:
I view this as non-blocking, and we can update the URL to be more specific in a future PR.

Reply:
@akeller, no, the converter accepts only BPMN file.  

If I remember correctly, you only have to change the `modeler:executionPlatform` and `modeler:executionPlatformVersion` values of a Camunda 7 DMN file to evaluate it on Camunda 8.

Reply:
I will add! Thank you!